v4/integration/web3/src/main/java/exchange/dydx/web3/EthereumInteractor.kt

```diff
@@ -88,9 +88,9 @@ class EthereumInteractor(
     ) {
         executor.submit {
             try {
-                val balance = web3j.ethGetBalance(address, DefaultBlockParameterName.LATEST).send()
+                val balance = web3j.ethGetBalance(address, DefaultBlockParameterName.LATEST).send().balance
                 runOnMainThread {
-                    completion(null, balance.balance)
```

Comment:
balance.balance can throw exceptions... so moving it outside runOnMainThread for the try/catch block to handle it.